apps/shared/avifjpeg.c

```diff
@@ -79,6 +247,25 @@ avifBool avifJPEGRead(const char * inputFilename, avifImage * avif, avifPixelFor
         avifImageSetProfileICC(avif, iccDataTmp, (size_t)iccDataLen);
     }
 
+    avif->yuvFormat = requestedFormat;
+    avif->depth = requestedDepth ? requestedDepth : 8;
+    // JPEG doesn't have alpha. Prevent confusion.
+    avif->alphaPremultiplied = AVIF_FALSE;
+
+    if (avifJPEGReadCopy(avif, &cinfo)) {
+        ret = AVIF_TRUE;
+        goto cleanup;
```

Comment:
For another PR of yours, I undid the pattern of "call a function that is only called once, otherwise `goto`". I think there are good usages for `goto` and single-use functions (certainly both exist in libavif), but single-use `goto` feels sloppy to me. Is there another way this can be done?

This won't block the PR (as I'll change it myself before accepting it), but as you are a regular contributor to libavif (thank you!), I wanted to let you know what I'm thinking as I review these.

Reply:
Done